### ISSUE TITLE
User becomes admin of room though not explicitly made one

### ISSUE THREAD

**Issue by khushman1** [vector-im/element-web#15008]
### Description

Faced this issue for the first time when I was kicking a user from multiple rooms in a community, and the community itself(they are no longer part of org.). User was granted admin rights to a particular room.

At first I thought it might have been a misclick on my part and I was cursing that UX person would think that putting a "make admin" button next to the "kick" button might be a good idea, but on further examination, it is clear that there is no way a misclick could have made them admin. At worst, they should have been banned or muted.

I am unable to remove them from the room anymore as they are an admin now (which is fine because I'm guessing it is a part of the design).

### Steps to reproduce

- No specific steps, was removing user from multiple rooms and on opening the right side panel on the user list for the particular user, the app suddenly promoted the user to admin.

Describe how what happens differs from what you expected.

- Expect to be able to kick the user, or at the very least, not make them admin :).

Logs being sent: yes

![element_admin_bug](https://user-images.githubusercontent.com/7322307/90689764-9098bc80-e23e-11ea-870c-33c4a5b895c6.png)

### Version information

- **Platform**: web (in-browser)

For the web app:

- **Browser**: Firefox 79.0 for Manjaro 1.0
- **OS**: Manjaro (Arch Linux base)
- **URL**: https://app.element.io/

**Comment by t3chguy** [comment]
The Matrix protocol does not allow Admins to kick other Admins to prevent the hell that IRC had with that issue.

Seems like they were made Admin prior to your attempt to change their permissions/kick them.

Nothing Element can do here.

What `make admin` button?

![image](https://user-images.githubusercontent.com/2403652/90751480-1748aa80-e2ce-11ea-88a3-9eea2b79d89b.png)


---

I advise you try and retry your steps in a test room as I could not manage to make someone an Admin.

The two ways to make someone an admin are:

`/op <MXID> <level>`

or Graphically:

![image](https://user-images.githubusercontent.com/2403652/90751621-465f1c00-e2ce-11ea-83da-62fb0633c544.png)
![image](https://user-images.githubusercontent.com/2403652/90751652-4eb75700-e2ce-11ea-9141-468b262f189d.png)
![image](https://user-images.githubusercontent.com/2403652/90751664-537c0b00-e2ce-11ea-93ff-6ae520cc0e91.png)

**Comment by khushman1** [comment]
Yeah, I'm not worried about not being able to kick them anymore. I expected it to be a part of the design, like in the original post.

Yes, **it is the action of them being made admin that was made erroneously by the Element web app, which is what this bug is about**. I know that I did not make them an Admin by mistake, as the UI is quite specific on the steps, i.e. click on their role, choose the different role from the dropdown, then press the green tick mark, and it seem improbable that I could have done all of those steps by mistake, especially because my intent was to press a button, i.e. kick which is far away from these options anyway.

Sorry about the "make admin" thing, it was just a joke. The button doesn't actually exist. I just thought initially when they became an admin, that I might have misclicked on the UI and converted their role into an admin, but obviously that couldn't have happened.

**Comment by t3chguy** [comment]
> Sorry about the "make admin" thing, it was just a joke. The button doesn't actually exist.

It used to :D

**Comment by khushman1** [comment]
> > Sorry about the "make admin" thing, it was just a joke. The button doesn't actually exist.
> 
> It used to :D

_shudders for admins like me with clumsy hands_ :P
